doc/userguide/install.rst

```diff
@@ -165,6 +165,34 @@ Follow these steps from your Suricata directory::
     make -j8 # j is for paralleling, you may de/increase depending on your CPU
     make install # to install your Suricata compiled binary
 
+
+Auto-Setup
+""""""""""
+
+You can also use the available auto-setup features of Suricata:
+
+::
+
+    ./configure && make && sudo make install-conf
+
+*make install-conf* would do the regular "make install" and then it would automatically
+create/setup all the necessary directories and ``suricata.yaml`` for you.
+
+::
+
+    ./configure && make && make install-rules
+
+*make install-rules* would do the regular "make install" and then it would automatically
+download and set up the latest ruleset from Emerging Threats available for Suricata.
+
+::
+
+    ./configure && make && make install-full
+
+*make install-full* would combine everything mentioned above (install-conf and install-rules)
+and will present you with a ready-to-run (configured and set-up) Suricata.
+
```

Comment:
I feel that `compilation` and `auto setup` should not be a part of the `Dependencies` section. Thoughts @jufajardini ?

Reply:
> I feel that `compilation` and `auto setup` should not be a part of the `Dependencies` section. Thoughts @jufajardini ?

I feel same too looking at it properly again, the dependencies I think are just prerequisite for the software to function which the steps are required to meet external software and library requirements but compilation and auto set up are part of the internal actions specific to the software install process. So likely the compilation and auto set-up should be in a different header section as still relating to the install process and not under dependencies. But then I still wait @jufajardini thoughts on it

Reply:
Good catch!


I'm thinking that... 
The `Compilation` Section shouldn't be a part of `Dependencies`, and changing that title would solve this. 
Might have been something that slipped in a past commit/review...
How does that look, to you?

Reply:
Actually, let's have two changes here:
- Make `Compilation` a section, instead of a subsection of `Dependencies`.
- Make `Auto Setup` another section. 

Seems to make more sense in my head, right now. Any thoughts you may have on that are welcome! :)

Reply:
I think the same too. I will make the changes